server/views/applications/pages/layout.njk

```diff
@@ -22,4 +38,4 @@
       </form>
     </div>
   </div>
-{% endblock %}
+{% endblock %}
```

Comment:
v minor: looks like there's a blank line missing at the end of this file

Reply:
hmmmm, v annoying. that's my new nunjucks formatter 👎